src/features/left-sidebar/LeftSidebar.md

```diff
@@ -241,7 +241,7 @@ const menuItems = [
     selected: false,
     showTooltip: true,
     iconComponent: IconFavorites,
-    showLoadingIndicator: true,
+    showLoadingIndicator: false,
```

Comment:
Is this intentional?

Reply:
Yes. Before the example shows favorites as loading. I think it would be more useful to display it normally.